### PR TITLE
docs(dql): with clause

### PR DESCRIPTION
**Pull request by aceforeverd**
- desc for with clause, supported since #2846 
- tweaks to online requirements.

### CODE REVIEW

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by zhanghaohit — APPROVED** [review]

File: docs/zh/openmldb_sql/deployment_manage/ONLINE_REQUEST_REQUIREMENTS.md [hunk 1]
```diff
@@ -58,11 +58,12 @@ SELECT substr(COL7, 3, 6) FROM t1;
 ### 在线请求模式下 `LAST JOIN` 的使用规范
 
 - 仅支持`LAST JOIN`类型。
-- 至少有一个JOIN条件是形如`left_table.column=right_table.column`的EQUAL条件，**并且`rgith_table.column`列需要命中右表的索引**。
-- 带排序LAST JOIN的情况下，`ORDER BY`只支持列表达式，**并且列需要命中右表索引的时间列**。
+- 至少有一个JOIN条件是形如`left_source.column=right_source.column`的EQUAL条件，**并且`rgith_source.column`列需要命中右表的索引（key 列）**。
```
Comment (dl239) [inline docs/zh/openmldb_sql/deployment_manage/ONLINE_REQUEST_REQUIREMENTS.md]: `right_source`

File: docs/en/reference/sql/dql/WITH_CLAUSE.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+| `SELECT` Statement Elements   | Offline Mode | Online Preview Mode | Online Request Mode | Note   |
+|:--------------------------|--------------|---------------------|---------------------|:-------|
+| WITH Clause                | **``✓``**    | **``✓``**           | **``✓``**           |  |
+
+## CTE rules
+
+1. CTEs can be referenced inside the query expression that contains the  `WITH`  clause.
+   This means the CTEs can be referenced by the outermost query contains the `WITH` clause,
+   as well as subqueries inside the outermost query.
+2. Each CTE in the same `WITH` clause must have a unique name.
```
Comment (dl239) [inline docs/en/reference/sql/dql/WITH_CLAUSE.md:31]: an

Comment (aceforeverd) [inline docs/en/reference/sql/dql/WITH_CLAUSE.md:31]: checked , 'a unique' is correct

**Review by dl239 — APPROVED** [review]

**Pull request merged**